library/core/crash_info/src/main/java/org/quiltmc/qsl/crash/mixin/SystemDetailsMixin.java

```diff
@@ -16,18 +16,15 @@
 
 package org.quiltmc.qsl.crash.mixin;
 
-import java.util.function.Supplier;
-
+import net.minecraft.util.SystemDetails;
+import org.quiltmc.qsl.crash.api.CrashReportEvents;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.Shadow;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
-import net.minecraft.util.SystemDetails;
-
-import net.fabricmc.loader.api.FabricLoader;
-import net.fabricmc.loader.api.ModContainer;
+import java.util.function.Supplier;
 
 @Mixin(SystemDetails.class)
 abstract class SystemDetailsMixin {
```

Comment:
Is the shadow of `#addSection` (which I can't comment on) used anywhere? If not I guess it can be removed